packages/create-permissionless-app/src/cli/index.ts

```diff
@@ -0,0 +1,150 @@
+    bundler: string
+    paymaster: string
+    signer: string
+    accountSystem: string
+}
+
+interface CliResults {
+    appName: string
+    flags: CliFlags
+}
+
+const defaultOptions: CliResults = {
+    appName: DEFAULT_APP_NAME,
+    flags: {
+        bundler: "pimlico",
+        paymaster: "pimlico",
+        signer: "privy",
+        accountSystem: "safe"
+    }
+}
+
+export const runCli = async (): Promise<CliResults> => {
+    const cliResults = defaultOptions
+
+    const program = new Command()
+        .name(CREATE_PERMISSIONLESS_APP)
+        .description("A CLI to spin up a permissionless app with Pimlico")
+        .argument(
+            "[dir]",
+            "The name of the application, as well as the name of the directory to create"
+        )
+
+        /** START CI-FLAGS
+         * As of now these CI Flags are not well structured but will be updated soon..
+         */
+        .option(
+            "-b, --bundler",
+            "Specify the bundler provider",
+            defaultOptions.flags.bundler
+        )
+        .option(
+            "-p, --paymaster",
+            "Specify the paymaster provider",
+            defaultOptions.flags.paymaster
+        )
+        .option(
+            "-s, --signer",
+            "Specify the signer",
+            defaultOptions.flags.signer
+        )
+        .option(
+            "-a, --account-system",
+            "Specify the account system",
+            defaultOptions.flags.accountSystem
+        )
+        /** END CI-FLAGS */
+        .parse(process.argv)
+
+    const cliProvidedName = program.args[0]
+    if (cliProvidedName) {
+        cliResults.appName = cliProvidedName
+    }
+
+    cliResults.flags = program.opts()
+
+    try {
+        const project = await p.group(
+            {
+                ...(!cliProvidedName && {
+                    name: () =>
+                        p.text({
+                            message: "What will your project be called?",
+                            defaultValue: cliProvidedName,
+                            validate: validateAppName
+                        })
+                }),
+                bundler: () => {
+                    return p.select({
+                        message: "Pick your bundler provider",
+                        options: [{ value: "pimlico", label: "Pimlico" }],
+                        initialValue: cliResults.flags.bundler
+                    })
+                },
+                paymaster: () => {
+                    return p.select({
+                        message: "Pick your paymaster provider",
+                        options: [{ value: "pimlico", label: "Pimlico" }],
+                        initialValue: cliResults.flags.paymaster
+                    })
+                },
+                accountSystem: () => {
+                    return p.select({
+                        message: "Pick your account system",
+                        options: [{ value: "safe", label: "Safe" }],
```

Comment:
I see you have simple account & kernel as well right?

Reply:
yep we have, was gonna add support for them later.. i ignored them since Kristof wanted to just have Safe for the v1 launch..